### PR TITLE
Move Notice string from PDF2 to base plug-in

### LINKED ISSUE #3888: Moving the "Notice" string from PDF2 to base
As suggested in https://github.com/dita-ot/dita-ot/issues/3882, I suggest to move the string `Notice` from PDF2 to the base plugin in all languages. The reason is, that all safety levels should be kept in the base plugin. Please let me know if you'd accept this change, then I'd send a PR.

### PR DESCRIPTION
**Pull request by jelovirt**
## Description
Move `Notice` string from PDF2 to base strings file.

## Motivation and Context
Fixes #3888

## How Has This Been Tested?
Existing tests.

## Type of Changes

- New feature _(non-breaking change which adds functionality)_

## Documentation and Compatibility
None needed

### CODE REVIEW

File: src/main/plugins/org.dita.base/xsl/common/strings-sl-si.xml [hunk 1]
```diff
@@ -33,6 +33,7 @@ See the accompanying LICENSE file for applicable license.
    <variable id="Value">Vrednost</variable>
    <variable id="Note">Opomba</variable>
    <variable id="Notes">Opombe</variable>
+   <variable id="Notice">Notice</variable><!--TODO:Notice-->
```
Comment (robander) [inline src/main/plugins/org.dita.base/xsl/common/strings-sl-si.xml:36]: In the original `sl.xml` this one was
`<variable id="Notice">Objava</variable>`

**Review by infotexture — COMMENTED** [review]
Shouldn't we target the `hotfix/3.7.1` branch for this and pull in @xephon2's latest **Notice** translations from #3882/#3883, which were already merged for 3.7.1?

That would allow these changes to merge cleanly, rather than modifying some of the same things on both `develop` and release branches.

If we're touching all of the **Notice** strings here, we might want to grab those ANSI Z535.4 localizations from #3889.

**Comment by jelovirt** [comment]
Will rebase on `hotfix/3.7.1`

**Comment by jelovirt** [comment]
The ANSI Z535.4 localizations from #3889 should be a separate feature PR. This refactor PR just moves variables to the correct files.

**Review by infotexture — APPROVED** [review]
👍 LGTM

**Pull request merged**